buildSrc/build.gradle.kts

```diff
@@ -14,3 +16,13 @@ dependencies {
 	implementation("com.github.jengelman.gradle.plugins:shadow:6.1.0")
 	implementation("org.gradle:test-retry-gradle-plugin:1.2.0")
 }
+
+kotlinDslPluginOptions {
+	experimentalWarning.set(false)
+}
```

Comment:
Any particular reason why this option is turned on? I understand it's to silence warnings about using experimental features of the Kotlin compiler, but it's not clear to me what warnings need to be silenced here.

Reply:
The `kotlin-dsl` plugin currently emits a warning message stating that it enables experimental features. Setting this option to `false` suppresses that warning. Please note that this will no longer be necessary with Gradle 7.0: https://github.com/gradle/gradle/pull/15460